runtime/ts/plan/suggestion.ts

```diff
@@ -53,22 +55,62 @@ export class Suggestion {
     return s2.rank - s1.rank;
   }
 
+  hasSearch(search) {
```

Comment:
I think we probably want to start adding types to input parameters of new functions.

Reply:
added